.github/workflows/python_bindings.yml

```diff
@@ -190,9 +197,8 @@ jobs:
         set -x
         cmake --build . --target package -j 2
 
+    # We always upload to testpypi
```

Comment:
Should we?

Reply:
I think so, for pushes to develop branch.